src/westmere/parser.c

```diff
@@ -22,7 +22,7 @@
 #include "generic/time.h"
 #include "generic/name.h"
 #include "generic/type.h"
-#include "generic/ip4.h"
+#include "westmere/ip4.h"
```

Comment:
Probably best to use this for the Haswell implementation too?

Reply:
I am not sure I understand your comment. I do use it for Haswell too, see `src/haswell/parser.c`. Of course, I implemented the function just one. Because the implementation works for westmere, I put it in the westmere header, but it is also included from haswell/parser.c.

Reply:
Sorry @lemire, oversight on my part